src/Doctrine/Odm/Paginator.php

```diff
@@ -27,29 +26,35 @@
  */
 final class Paginator implements \IteratorAggregate, PaginatorInterface, HasNextPagePaginatorInterface
 {
-    public const LIMIT_ZERO_MARKER_FIELD = '___';
-    public const LIMIT_ZERO_MARKER = 'limit0';
```

Comment:
This constants are removed. They should have been marked as internal. We can keep them for backward compatibility, but they are useless now that they are not used in the aggregation pipeline.